<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>631</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_normal">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>120</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Noto Mono&quot;;
border:2px groove gray;
padding:2px 4px;
background-color: rgb(220, 220, 220);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>快速开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rank">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>220</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Noto Mono&quot;;
background-color: rgb(220, 220, 220);
border:2px groove gray;
border-radius:10px;
padding:2px 4px;
</string>
    </property>
    <property name="text">
     <string>排位匹配</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_create_room">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>320</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Noto Mono&quot;;
background-color: rgb(220, 220, 220);
border:2px groove gray;
border-radius:10px;
padding:2px 4px;
</string>
    </property>
    <property name="text">
     <string>创建房间</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_join_room">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>410</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Noto Mono&quot;;
background-color: rgb(220, 220, 220);
border:2px groove gray;
border-radius:10px;
padding:2px 4px;
</string>
    </property>
    <property name="text">
     <string>加入房间</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_rid">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>500</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>输入加入的房间号（仅限数字）</string>
    </property>
   </widget>
   <widget class="QListWidget" name="RankList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>477</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lable_user_picture">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/image/avatar04.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Noto Mono&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>全服排名</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_username">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>22</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 17pt &quot;Noto Mono&quot;;</string>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lable_points">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>22</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Mono</family>
      <pointsize>17</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 17pt &quot;Noto Mono&quot;;</string>
    </property>
    <property name="text">
     <string>积分：2000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lable_uid">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>22</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 17pt &quot;Noto Mono&quot;;</string>
    </property>
    <property name="text">
     <string>uid:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>934</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
